Celeste.Mod.mm/Patches/Session.cs

```diff
@@ -25,6 +99,40 @@ public void ctor(AreaKey area, string checkpoint = null, AreaStats oldStats = nu
             }
             orig_ctor(area, checkpoint, oldStats);
         }
+
+        public float GetSlider(string slider) {
+            if (_Sliders.TryGetValue(slider, out Slider obj)) {
+                return obj._Value;
+            }
+            return 0f;
+        }
+
+        public Slider GetSliderObject(string slider) {
+            if (!_Sliders.TryGetValue(slider, out Slider obj)) {
+                _Sliders[slider] = obj = new(this, slider, 0f);
+                Everest.Events.Session.SliderChanged(obj, null);
+            }
+            return obj;
+        }
+
+        public void SetSlider(Slider slider, float value) {
+            float previous = slider._Value;
+            slider._Value = value;
+            Everest.Events.Session.SliderChanged(slider, previous);
```

Comment:
This is a bit weird, maybe I'd move the event firing code to the Slider.Value setter instead, then this method doesn't even have to exist (If you have a Slider already, then you have the Value setter publicly available)

Reply:
true